ui/app/AppLayouts/Profile/Sections/AdvancedContainer.qml

```diff
@@ -163,6 +163,15 @@ Item {
             }
         }
 
+        StatusSettingsLineButton {
+            text: "GIF"
```

Comment:
`GIF search` or `GIF widget` might be better, since `GIF` just sounds like enabling normal gifs, which is not it. Then  again, I know you didn't have a design, so if you prefer to have a design first, it's ok